likeminds-chat-reactnative-ui/components/LMChatButton/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import React, { useState } from "react";
+import { LMChatIcon } from "../LMChatIcon";
+import Styles from "../constants/Styles";
+import { LMChatButtonProps } from "./types";
+
+export const LMChatButton = ({
+  text,
+  icon,
+  onTap,
+  placement,
+  isActive,
+  activeIcon,
+  activeText,
```

Comment:
what do these props mean?

Reply:
these props are mainly to show `LMChatIcon` in different ways